src/pyTRLCConverter/rst_converter.py

```diff
@@ -0,0 +1,701 @@
+
+        # For proper reStructuredText formatting, the first written part shall not have an empty line before.
+        # But all following parts (heading, table, paragraph, image, etc.) shall have an empty line before.
+        # And at the document bottom, there shall be just one empty line.
+        self._empty_line_required = False
+
+    @staticmethod
+    def get_subcommand() -> str:
+        # lobster-trace: SwRequirements.sw_req_rst
+        """
+        Return subcommand token for this converter.
+
+        Returns:
+            str: Parser subcommand token
+        """
+        return "rst"
+
+    @staticmethod
+    def get_description() -> str:
+        # lobster-trace: SwRequirements.sw_req_rst
+        """
+        Return converter description.
+
+        Returns:
+            str: Converter description
+        """
+        return "Convert into reStructuredText format."
+
+    @classmethod
+    def register(cls, args_parser: any) -> None:
+        # lobster-trace: SwRequirements.sw_req_rst_multiple_doc_mode
+        # lobster-trace: SwRequirements.sw_req_rst_single_doc_mode
+        # lobster-trace: SwRequirements.sw_req_rst_sd_top_level_default
+        # lobster-trace: SwRequirements.sw_req_rst_sd_top_level_custom
+        # lobster-trace: SwRequirements.sw_req_rst_out_file_name_default
+        # lobster-trace: SwRequirements.sw_req_rst_out_file_name_custom
+        """
+        Register converter specific argument parser.
+
+        Args:
+            args_parser (any): Argument parser
+        """
+        super().register(args_parser)
+
+        BaseConverter._parser.add_argument(
+            "-e",
+            "--empty",
+            type=str,
+            default=BaseConverter.EMPTY_ATTRIBUTE_DEFAULT,
+            required=False,
+            help="Every attribute value which is empty will output the string." \
+                f"(default = {BaseConverter.EMPTY_ATTRIBUTE_DEFAULT})."
+        )
+
+        BaseConverter._parser.add_argument(
+            "-n",
+            "--name",
+            type=str,
+            default=RstConverter.OUTPUT_FILE_NAME_DEFAULT,
+            required=False,
+            help="Name of the generated output file inside the output folder " \
+                f"(default = {RstConverter.OUTPUT_FILE_NAME_DEFAULT}) in " \
+                "case a single document is generated."
+        )
+
+        BaseConverter._parser.add_argument(
+            "-sd",
+            "--single-document",
+            action="store_true",
+            required=False,
+            default=False,
+            help="Generate a single document instead of multiple files. The default is to generate multiple files."
+        )
+
+        BaseConverter._parser.add_argument(
+            "-tl",
+            "--top-level",
+            type=str,
+            default=RstConverter.TOP_LEVEL_DEFAULT,
+            required=False,
+            help="Name of the top level heading, required in single document mode. " \
```

Comment:
Here we end in ". " but append the default.  
Align on one wording for all arguments. 

Recommend to only " " or ", " before the default